cli/command/orchestrator.go

```diff
@@ -53,9 +55,14 @@ func normalize(value string) (Orchestrator, error) {
 	}
 }
 
+// NormalizeOrchestrator parses an orchestrator value and check if it is valid
+func NormalizeOrchestrator(value string) (Orchestrator, error) {
```

Comment:
Looks like it's unused? 🤔

Reply:
Used by context subcommands (`docker context create`)

Reply:
Ok it's used in the follow up #1501

Reply:
Ah, yes; also noticed it wasn't used 😂